deps/build.jl

```diff
@@ -16,6 +28,9 @@ if Base.BLAS.vendor() === :mkl
             const libfftwf = "$mklpath"
         """)
     end
+elseif provider == "MKL"
+    error("MKL build requested for FFTW but Julia was not built with MKL.\n",
+          "To fix this, " action, " and rerun Pkg.build(\"FFTW\").")
```

Comment:
With my above suggestion, you just need to set `ENV["JULIA_FFTW_PROVIDER"]="FFTW"` and it will work.